### ISSUE TITLE
false positive - hopp.bio

### ISSUE THREAD

**Issue by shaulgo** [nextdns/metadata#1316]
Hey,

Hopp.bio is a link in bio by Wix.com. It is erroneously flagged by Antiy-AVL on Virustotal. I tried contacting them multiple times. But they are not answering. 

Could you please remove it from your blocklist? If it originates from another blocklist please tell me where it is coming from.

<img width="1023" alt="image" src="https://github.com/nextdns/metadata/assets/2996028/4a03e53b-6be7-4350-92d8-e497377f17ba">

**Comment by shaulgo** [comment]
@hagezi any chance you can take a look 🙏

**Comment by hagezi** [comment]
I have no opportunity to remove this from NextDNS TIF. The domain is currently also blocked by DNS0.eu.
```
Secure DNS:
 - 360Secure       OK
 - AliDNS          OK
 - CFIEC           OK
 - CleanBrowsing   OK
 - Cloudflare      OK
 - ComodoSecure    OK
 - CONTROLD.TIF    OK
 - DNS0.eu         BLOCKED
 - DNS0.eu.ZERO    BLOCKED
 - DNSWatchGO      OK
 - HaGeZi.TIF      OK
 - Neustar         OK
 - NextDNS.TIF_AI  BLOCKED
 - NortonCS        OK
 - NRD.DGA.IDN     OK
 - Quad9           OK
 - SafeDNS         BLOCKED
 - UltraDNS        OK
 - Umbrella        OK
 - YandexSafe      OK

Blocklists:
 - 1Hosts.Lite     OK
 - 1Hosts.Mini     OK
 - 1Hosts.Pro      OK
 - AdGuardDNS      OK
 - AhaDNS          OK
 - CONTROLD.AT     OK
 - DevDansHosts    OK
 - EasyList        OK
 - GoodbyeAds      OK
 - HaGeZi.LIGHT    OK
 - HaGeZi.NORMAL   OK
 - HaGeZi.PRO      OK
 - HaGeZi.PRO.PLUS OK
 - HaGeZi.TIF      OK
 - HaGeZi.ULTIMATE OK
 - hBlock          OK
 - NextDNS.AT      OK
 - OISD.Big        OK
 - OISD.Small      OK
 - QuidsUp.NOTRACK OK
 - StevenBlack     OK
```

ping @rs @romaincointepas

**Comment by shaulgo** [comment]
Thanks I have reached out to DNS0.eu. Once they fix it, will it be unblocked from nextDNS automatically?

**Comment by hagezi** [comment]
I don't think so, I don't think they use the same database in NextDNS. At least there are blocked domains in DNS0.eu that are not blocked by the NextDNS TIF and vice versa.

**Comment by shaulgo** [comment]
oh, I understood from your comment that you can't remove it because it's blocked there as well.
But you were just letting me know that it's blocked there as well?
How can I unblock it from NextDNS TIF?

**Comment by hagezi** [comment]
Why do so many people think I am NextDNS support?

I have nothing to do with NextDNS, I don't have any authorisations, the operators have to do that themselves, so I can't remove it.

The domains are not blocked in my lists.

Operators: @rs @romaincointepas

**Comment by shaulgo** [comment]
I just looked at a couple of random issues and saw you answering. 
Thank you very much for letting me know about DNS0.eu. And for all your help 🙏

**Comment by hagezi** [comment]
Maybe this will help you, the domain is flagged by Phishtank:

![grafik](https://github.com/nextdns/metadata/assets/104343908/28462d7d-957a-4a02-ba0b-fbfa2d970025)

https://www.phishtank.com/phish_detail.php?phish_id=8409966
https://www.urlvoid.com/scan/hopp.bio/

Phish: `https://hopp[.]bio/messageee`

Otherwise I have not found any feeds in which the main domain is flagged.

**Comment by emidaniel** [comment]
So, let's place another one of these degenerate "AI" bots on our abuse desk and block abuse reports to ourselves with it, then wonder why our domains are subjects to carpet blocks like this. Really no idea where it could have gone wrong!

![image](https://github.com/nextdns/metadata/assets/157081461/a95eaf06-0a7f-4a41-adaf-818e61cd6a02)

**Comment by shaulgo** [comment]
I'm not sure what you mean?

**Comment by emidaniel** [comment]
Isn't it clear from the screenshot?

Hopp.to is your domain, and when I try to report a phish (`http://bovenkamer.hopp.to/contact`) to you through the website (hopp.to), I'm redirected to hopp.co, then to https://www.wix.com/abuse ("Report phish" link at the bottom), and the bot you set up there ignores my report and falsely claims that "this site is not hosted by Wix" because it is not configured to accept reports for all the domains you own.

This is not the first time there have been issues like that one. When it becomes too hard to deal with this, some people (and me specifically) just give up and submit links to multiple blacklists (phishtank, netcraft, AVs, DNS blocklists, and so on), which often causes the entire domain to be blocked. Then you remove the phish and try to delist the domain. But after some time another abuse report is ignored and domain gets flagged again...

**Comment by shaulgo** [comment]
WOW, I'm so sorry this is terrible. Hopp is sort of a "startup" inside of Wix. I wasn't aware of this. We will fix it asap.
By the way, we are constantly working on improving our handling of phishing. Our team didn't have experience in this area and I couldn't find good information online. So we are continuing to improve as we learn how to better handle it.
I was on vacation the last couple of days. I recived your first message in my email without the image. So I didn't understand it.
I want to thank you for bringing this up and caring. If there is anything else you are aware of. It will be great if you can share.
Like if there is some protocol that other companies use for better/easier reporting?

**Comment by shaulgo** [comment]
WOW, I'm so sorry. This is terrible. 
Hopp.bio is a sort of "startup" inside of Wix. I'm working on fixing this issue asap.

I apologize for the delayed response. I was on vacation and initially received your message, through email without the image, which led to my misunderstanding. 

We're actively enhancing our phishing prevention measures as our team gains expertise in this area.

I want to thank you for bringing this up and caring. If you have any suggestions for best practices or improvements, I'll appreciate your insights.
